packages/vats/tools/bootstrap-chain-reflective.js

```diff
@@ -16,6 +16,16 @@ import * as basicBehaviorsNamespace from '../src/core/basic-behaviors.js';
 import * as chainBehaviorsNamespace from '../src/core/chain-behaviors.js';
 import * as utils from '../src/core/utils.js';
 
+/**
+ * @typedef {Record<
+ *   | 'BASIC_BOOTSTRAP'
+ *   | 'CHAIN_BOOTSTRAP'
+ *   | 'MINIMAL'
+ *   | 'SHARED_CHAIN_BOOTSTRAP',
+ *   import('@agoric/vats/src/core/lib-boot.js').BootstrapManifest
+ * >} BootstrapManifests
```

Comment:
This seems overly elaborate; we just need the strings.
```suggestion
 * @typedef {'BASIC_BOOTSTRAP' | 'CHAIN_BOOTSTRAP' | 'SHARED_CHAIN_BOOTSTRAP' | 'MINIMAL'} BootstrapManifestName
```
and then optionally also:
```patch
 const {
   CHAIN_BOOTSTRAP_MANIFEST: CHAIN_BOOTSTRAP,
   SHARED_CHAIN_BOOTSTRAP_MANIFEST: SHARED_CHAIN_BOOTSTRAP,
   ...chainBehaviors
 } = chainBehaviorsNamespace;
+/** @type {Record<BootstrapManifestName, import('@agoric/vats/src/core/lib-boot.js').BootstrapManifest>} */
 const manifests = { BASIC_BOOTSTRAP, CHAIN_BOOTSTRAP, SHARED_CHAIN_BOOTSTRAP };
 const allBehaviors = { ...basicBehaviors, ...chainBehaviors };
 export const modules = {
   behaviors: { ...allBehaviors },
   utils: { ...utils },
 };
```
```patch
 /**
  * @param {VatPowers & { D: DProxy; testLog: typeof console.log }} vatPowers
  * @param {{
- *   baseManifest?: string;
+ *   baseManifest?: BootstrapManifestName;
  *   addBehaviors?: string[];
  *   coreProposalCodeSteps?: string[];
  * }} bootstrapParameters
  * @param {import('@agoric/vat-data').Baggage} baggage
  */
 export const buildRootObject = (vatPowers, bootstrapParameters, baggage) => {
```